string_manipulation.rb

```diff
@@ -208,7 +218,10 @@ def reverse_sentence(my_sentence)
   return my_sentence
```

Comment:
Similar to the first two methods, this method also updates the input string in place and hence you don't need to return anything.